packages/graph-explorer/src/core/ConfigurationProvider/types.ts

```diff
@@ -142,9 +142,14 @@ export type ConnectionConfig = {
   awsRegion?: string;
   /**
    * Number of milliseconds before aborting a request.
-   * By default, 60 seconds.
+   * By default, undefined.
    */
   fetchTimeoutMs?: number;
```

Comment:
is there default limit for nodejs fetch?

Reply:
The native fetch timeouts are browser specific. This value is setting an artificial timeout of the user's choosing.

I changed the comment here because there is no default. New connections do not have a timeout value set by default.